### PR TITLE
Remove family from alerts

### PR DESCRIPTION
**Pull request by MrZammler**
##### Summary

This PR removes the `family` attribute from alerts. This includes the configuration, silencers api, and communication with the cloud.

The entry remains in the sqlite database so as to not cause issues when the user downgrades.

##### Test Plan

Most of the alert functionality should be tested, including notifications, and communication with the cloud.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by thiagoftsm** [comment]
Hello @MrZammler ,

I ran next command on your branch:

```sh
$ grep -r family health/*| less
```

and I observed that we still have information abou `family` in:

- health/REFERENCE.md
- health/notifications/health_alarm_notify.conf
- health/notifications/custom/metadata.yaml
- health/notifications/custom/README.md

Please, take a look on them.

Except for these files, the PR is working as expected.

File: health/notifications/awssns/README.md [hunk 1]
```diff
@@ -92,8 +92,6 @@ The supported variables are:
 | `${url_name}`               | Same as ${name} but URL encoded                                                  |
 | `${chart}`                  | The name of the chart (type.id)                                                  |
 | `${url_chart}`              | Same as ${chart} but URL encoded                                                 |
-| `${family}`                 | The family of the chart                                                          |
```
Comment (ilyam8) [inline health/notifications/awssns/README.md:95]: @Ancairon these notification readmes are generated, right? If so, we shouldn't upate them manually @MrZammler

**Comment by ilyam8** [comment]
@MrZammler please check if there is something you need to update too

```bash
$ git grep families -- ':(exclude)*.svg' ':(exclude)*.md' ':(exclude)*.js' ':(exclude)*.map'
aclk/schema-wrappers/alarm_config.h:    char *families;
collectors/python.d.plugin/pandas/metadata.yaml:              description: "[family](https://learn.netdata.cloud/docs/data-collection/chart-dimensions-contexts-and-families#family) of the chart to be displayed in the dashboard."
collectors/python.d.plugin/pandas/metadata.yaml:              description: "[context](https://learn.netdata.cloud/docs/data-collection/chart-dimensions-contexts-and-families#context) of the chart to be displayed in the dashboard."
database/contexts/api_v2.c:            buffer_json_member_add_string(wb, "families", t->selectors.families);
database/contexts/rrdcontext.h:        const char *families;
database/rrd.h:    DICTIONARY *rrdfamily_root_index;               // the host's chart families index
database/sqlite/sqlite_functions.c:    "every text, units text, calc text, families text, plugin text, module text, charts text, green text, "
database/sqlite/sqlite_health.c:    " units, calc, families, plugin, module, charts, green, red, warn, crit, "                                         \
database/sqlite/sqlite_health.c:        acd.selectors.families = (const char *) sqlite3_column_text(res, param++);
libnetdata/socket/socket.c:    sockets->fds_families[sockets->opened] = family;
libnetdata/socket/socket.h:    int fds_families[MAX_LISTEN_FDS];   // the family of the open sockets (AF_UNIX, AF_INET, AF_INET6)
libnetdata/xxhash.h: * It is proposed in four flavors, in three families:
libnetdata/xxhash.h: * @see @ref XXH64_family, @ref XXH3_family : Other xxHash families
tests/health_mgmtapi/expected_list/FAMILIES_LOAD-list.json:{ "all": false, "type": "None", "silencers": [ { "families": "load" } ] }
tests/health_mgmtapi/expected_list/SILENCE_2-list.json:{ "all": false, "type": "SILENCE", "silencers": [ { "families": "load" } ] }
tests/health_mgmtapi/health-cmdapi-test.sh.in:  cmd "families=load" "${resp}"
web/api/netdata-swagger.json:            "name": "families",
web/api/netdata-swagger.json:            "description": "The alarm families.",
web/api/netdata-swagger.json:            "description": "The family of the context. When multiple charts of a context have different families, the netdata server replaces the different parts with [x], so that the context can have only one family."
web/api/netdata-swagger.yaml:        - name: families
web/api/netdata-swagger.yaml:          description: The alarm families.
web/api/netdata-swagger.yaml:          description: "The family of the context. When multiple charts of a context have different families,
```

File: health/notifications/awssns/README.md (hunk 1 above)
Comment (Ancairon) [reply health/notifications/awssns/README.md:95]: Indeed it is, look at the note at the top of the file https://github.com/netdata/netdata/pull/16025/files#diff-8cc0362d13426093c886b89dd5b6f4465081c299062c81436df3c8696435ab41L7

Comment (Ancairon) [reply health/notifications/awssns/README.md:95]: @MrZammler you need to edit the yaml file for the changes to take effect.

Comment (MrZammler) [reply health/notifications/awssns/README.md:95]: Ah, didn't see that, sorry. I've also edited the yaml file as part of this PR, should I revert the README back, or it's not a problem?

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]
Last commits did not affect codes, so  I am approving without retest.

**Pull request merged**